modules/prompt/functions/prompt_sorin_setup

```diff
@@ -38,8 +38,10 @@ function prompt_sorin_pwd {
   if [[ "$pwd" == (#m)[/~] ]]; then
     _prompt_sorin_pwd="$MATCH"
     unset MATCH
-  else
+  elif [[ "$SORIN_PROMPT_SHORT_PWD" == 'Y' ]]; then
```

Comment:
I very much like this, but I think the condition should be based off `zstyle` instead of environment variables, so that we don't muddy up the environment unnecessarily, and so that zprezto configuration stays inside your zpreztorc file.

Something more like:
```
elif zstyle -t ':prezto:module:prompt:theme:sorin' shortPWD; then
  _prompt_sorin_pwd="${${${${(@j:/:M)${(@s:/:)pwd}##.#?}:h}%/}//\%/%%}/${${pwd:t}//\%/%%}"
fi
```